src/utils/index.ts

```diff
@@ -0,0 +1,28 @@
+import axios, { AxiosRequestConfig } from "axios";
+
+// note: not used, but could be used with GET with params
+export async function getData(url: string, params?: AxiosRequestConfig) {
+  try {
+    const res = await axios.get(url, params);
+    const data = await res.data;
+    return data;
+  } catch (error) {
+    console.log(error, `error - getData in ${url} route`);
+  }
+}
+
+export async function getAllData(url: string) {
+  try {
+    const res = await axios.get(url);
+    const data = await res.data;
+    return data;
+  } catch (error) {
+    console.log(error, `error - getAllData in ${url} route`);
+
+    if (axios.isAxiosError(error)) {
+      return {
+        data: "API request failed! Did you remember to `npm start` the backend app?",
+      };
+    }
```

Comment:
Converted these functions to typescript as well as hinting to start the backend app if there was an error

Reply:
Heads up that there's a small typo here -- `npm run dev` starts the backend app, rather than `npm start`.

Reply:
fixed, thank you!